--- .bashrc.ORIG	2010-04-19 03:15:58.000000000 +0100
+++ .bashrc	2011-03-07 15:39:00.000000000 +0000
@@ -36,7 +36,7 @@
 # uncomment for a colored prompt, if the terminal has the capability; turned
 # off by default to not distract the user: the focus in a terminal window
 # should be on the output of commands, not on the prompt
-#force_color_prompt=yes
+force_color_prompt=yes
 
 if [ -n "$force_color_prompt" ]; then
     if [ -x /usr/bin/tput ] && tput setaf 1 >&/dev/null; then
@@ -50,7 +50,20 @@
 fi
 
 if [ "$color_prompt" = yes ]; then
-    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+    red="0;31";    redl="1;31"; 
+    green="0;32";  greenl="1;32"
+    purple="0;35"; purplel="1;35"
+    cyan="0;36";   cyanl="1;36"
+    blue="0;34";   bluel="1;34"
+                   yellowl="1;33"
+
+    hostname -s | grep -q "\-live[0-9]*$"    && prompt_color=${redl}
+    hostname -s | grep -q "\-staging[0-9]*$" && prompt_color=${yellowl}
+    hostname -s | grep -q "\-uat[0-9]*$"     && prompt_color=${greenl}
+    hostname -s | grep -q "\-dev[0-9]*$"     && prompt_color=${cyanl}
+
+   #PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+    PS1='${debian_chroot:+($debian_chroot)}\u@\[\033[${prompt_color}m\]\h\[\033[00m\]:\[\033[${prompt_color}m\]\w\[\033[00m\]\$ '
 else
     PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
 fi
